lib/netuitive_rails_agent/action_controller.rb

```diff
@@ -49,17 +49,20 @@ def process_action(*args)
         event = ActiveSupport::Notifications::Event.new(*args)
         controller = event.payload[:controller].to_s
         action = event.payload[:action].to_s
-        interaction.add_sample("action_controller.#{controller}.#{action}.request.total_duration", event.duration)
-        interaction.add_sample("action_controller.#{controller}.#{action}.request.query_time", event.payload[:db_runtime])
-        interaction.add_sample("action_controller.#{controller}.#{action}.request.view_time", event.payload[:view_runtime])
-        interaction.add_sample("action_controller.#{controller}.request.total_duration", event.duration)
-        interaction.add_sample("action_controller.#{controller}.request.query_time", event.payload[:db_runtime])
-        interaction.add_sample("action_controller.#{controller}.request.view_time", event.payload[:view_runtime])
+        ac_whitelist_string = NetuitiveRailsAgent::ConfigManager.action_controller_whitelist
+        if controller =~ /#{ac_whitelist_string}|test_controller/
```

Comment:
Why explicitly allow the `test_controller`?

Reply:
Because we forgot to remove that after fixing the tests...